content/docs/reference/tracing.mdx

```diff
@@ -16,109 +16,136 @@ import TabItem from '@theme/TabItem';
 
 ## Summary
 
-Tracing tracks the progression of a single user request as it is handled by Pomerium.
+Pomerium has comprehensive support for OpenTelemetry tracing, allowing detailed introspection into requests and authorization flows. You can use tracing to debug errors and latency issues in your applications.
 
-Each unit of work is called a Span in a trace. Spans include metadata about the work, including the time spent in the step (latency), status, time events, attributes, links. You can use tracing to debug errors and latency issues in your applications, including in downstream connections.
-
-## How to configure
+## Configuration
 
 <Tabs>
 <TabItem value="Core" label="Core">
 
-#### Shared Tracing Settings
+### Environment Variables
+
+The recommended way to configure tracing is by using the standard OpenTelemetry environment variables:
+
+- [SDK environment variables](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#general-sdk-configuration)
+- [OTLP exporter environment variables](https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/)
+
+The main variables used to configure tracing in Pomerium are the following:
+
+| Name | Description | Default |
+| :-- | :-- | :-- |
+| [`OTEL_TRACES_EXPORTER`](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#exporter-selection) | Trace exporter to be used. <br/> Valid values are `"otlp"` or `"none"` | `"none"` |
+| [`OTEL_EXPORTER_OTLP_ENDPOINT`](https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/#otel_exporter_otlp_endpoint) or <br/> [`OTEL_EXPORTER_OTLP_TRACES_ENDPOINT`](https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/#otel_exporter_otlp_traces_endpoint) | See [Endpoint Configuration](https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/#endpoint-configuration). |
```

Comment:
For the description, include a short description or example values and then the external link that you have now is fine below that as "For more details, see the OTEL documentation.", but we shouldn't send people external for basic info.